client/src/app/app.component.html

```diff
@@ -4,7 +4,7 @@
       <div class="logo">
         <h1>ODS Provisioning</h1>
       </div>
-      <div class="user-area" [hidden]="router.url.indexOf('/logout') !== -1">
+      <div class="user-area" [hidden]>
```

Comment:
Why does ``hidden`` have ``[...]`` around and will it then be always hidden?

Reply:
Oops, of course only `hidden` is needed here.